### PR TITLE
labels: defer resolving labels

### PR DESCRIPTION
**Pull request by sejoker**
These changes defer the process of label resolving
to prevent 404 errors from Github API

Closes https://github.com/nodejs/github-bot/issues/79

### CODE REVIEW

File: lib/node-repo.js [hunk 1]
```diff
@@ -4,6 +4,15 @@ const githubClient = require('./github-client')
 
 const resolveLabels = require('./node-labels').resolveLabels
 
+function deferredResolveLabelsThenUpdatePr (options) {
+  if (options.timeoutInSec) {
+    setTimeout(() => resolveLabelsThenUpdatePr(options),
+              options.timeoutInSec * 1000)
+  } else {
+    resolveLabelsThenUpdatePr(options)
```
Comment (phillipj) [inline lib/node-repo.js]: What if we got rid of this `else` block and always executed this async? Something like:

```javascript
const timeoutMillis = (options.timeoutInSec || 0) * 1000

setTimeout(resolveLabelsThenUpdatePr, timeoutMillis, options)
```

**Comment by phillipj** [comment]
Little nitpick, other than that it LGTM

File: lib/node-repo.js (hunk 1 above)
Comment (sejoker) [inline lib/node-repo.js]: @phillipj updated PR as per comments, looks cleaner, thanks.

**Comment by phillipj** [comment]
Awesome, thanks for contributing! 👍

**Pull request merged**